code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -307,8 +307,13 @@
 		/obj/item/clothing/gloves,
 		/obj/item/device/assembly/mousetrap,
 		/obj/item/weapon/crowbar/prybar,
-		/obj/item/clothing/mask/plunger
-		)
+		/obj/item/clothing/mask/plunger,
+		/obj/item/weapon/storage/bag/trash,
+		/obj/item/weapon/storage/bag/trash/bluespace,
```

Comment:
As Purple already commented, this isn't the best idea.

Reply:
Remove them and it should be fine.

Reply:
fixed